mock/account.go

```diff
@@ -0,0 +1,77 @@
+package mock
+
+import (
+	"math/big"
+)
+
+// Account holds the account info
+type Account struct {
```

Comment:
you already defined this in ipc/common.go

Reply:
The current mocks depends on this - and the struct has some mock-related features. Can we keep it like this? Fixed by renaming to `AccountMock`.